### ISSUE TITLE
addCue() does not return an ID

### ISSUE THREAD

**Issue by wktdev** [processing/p5.js-sound#297]
I don't know if this is a bug or if I am simply misreading the documentation. When I do the following addCue does not return an "ID" per the documentation, it returns NaN. 

```
  mySound.play();
  let id = mySound.addCue(3, console.log,"we are the world");

  console.log(id); // NaN
```

**Comment by XeniousSoftware** [comment]
Yes this is actually an issue and i know how to fix it. i'll submit a PR with the fix.

**Comment by therewasaguy** [comment]
should be fixed with #298 - thanks @XeniousSoftware !
